docs/src/pages/index.md

```diff
@@ -5,16 +5,15 @@ permalink: /index.html
 
 Welcome to our documentation pages! What would you like to view?
 
-## [User Guide](user-guide/)
+## [User Guide](use/)
 
 Intended for end users of ESLint. Contains information about core rules, configuration, command line options, formatters, and integrations,
 as well as guides for migrating from earlier versions of ESLint.
 
-## [Developer Guide](developer-guide/)
+## [Developer Guide](extend/)
```

Comment:
Since we're changing the link and the description, maybe we should change the link text too?
```suggestion
## [Extender Guide](extend/)
```

Reply:
was going to do this as next task as part of https://github.com/eslint/eslint/issues/16720